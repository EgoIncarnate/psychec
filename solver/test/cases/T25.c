int main()
{
  inteiro x = 2;
  real y = 3.0;
  printf("multiplicando %d e %.2f ", x, y);
  printf("temos:\n");
  printf("resposta: %f\n", x * y);
  
  return 0;
}
