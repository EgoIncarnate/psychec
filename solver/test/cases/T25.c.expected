typedef double real ;
typedef int inteiro ;
